editors/sc-ide/widgets/code_editor/editor.hpp

```diff
@@ -41,10 +41,19 @@ class GenericCodeEditor : public QPlainTextEdit {
 
 public:
     GenericCodeEditor(Document*, QWidget* parent = NULL);
+    ~GenericCodeEditor();
+
+    // Rule of 5 -- with an explicit dtor, make sure that copy ctor/assign and move
+    // ctor/assign are deleted.
+    // If these must be implemented in the future, make sure they take care of
+    // mDoc->lastActiveEditor() appropriately. See source of dtor method.
+    GenericCodeEditor(GenericCodeEditor& other) = delete;
+    GenericCodeEditor& operator=(GenericCodeEditor& other) = delete;
```

Comment:
you dropped `const` from the copy ops when you moved them -- can you please add those back?